CHANGELOG.md

```diff
@@ -1,5 +1,125 @@
 # Changelog
 
+## [0.59]
+
+Welcome to release 0.59 of React Native! For highlights of this release, please view the dedicated [blog post](#todo). Thanks to those who gave feedback on during the [release candidate phase]([https://github.com/react-native-community/react-native-releases/issues/41](https://github.com/react-native-community/react-native-releases/issues/79)). If you're interested in helping evaluate our next release (0.60), subscribe to the dedicated issue [here](https://github.com/react-native-community/react-native-releases/issues/#todo).
+
+### Added
+
+- Add a Metro configuration to the template with inline require/import options ([ae11993](https://github.com/facebook/react-native/commit/ae11993) by [@cpojer](https://github.com/cpojer))
```

Comment:
@cpojer do we want to highlight the purpose here and ask people to test and give feedback? Should we link to the performance part of the blog that talks about this?

@turnrye Are we talking about highlights at all or is that only going in the blog post?

Reply:
I'd like to push folks to the blogpost for highlights and get the changelog focused on just the changes. Long term I think this positions us to have the changelog be more advanced (considering folks who are creating libs for instance) and then the blogpost be what the general public reviews.